### ISSUE TITLE
Price filter by proof id

### ISSUE THREAD

**Issue by raphodn** [openfoodfacts/open-prices#218]
### Story

In the frontend, it would be useful for users to fetch prices corresponding to a specific proof

**Comment by Sudhanva-Nadiger** [comment]
#160 is same as this issue right?

**Comment by raphodn** [comment]
Oh yeah you're right 😅😅

@raphael0202 is there a privacy issue actually or not ? I think I might have got confused here, between private proofs because of personal data on the image vs private prices (which is not the case here).

Ok to add a proof_id filter on Price ? And be able to view/add new prices for a given proof ?

**Comment by raphael0202** [comment]
> Ok to add a proof_id filter on Price ? And be able to view/add new prices for a given proof ?

Yes that's fine, by contributing to Open Prices you already agree that your consumption data can be extracted from it (if you choose to upload receipts). Only receipt images with personal information are problematic and should be marked as private (`is_public=False`)

**Comment by raphodn** [comment]
ok so in the end @Sudhanva-Nadiger it's simpler :sweat_smile: 
sorry for having done #220
